kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -774,16 +774,8 @@ define([
         },
   
         initImport: function() {
-            let stagingAreaViewer = this; 
-  
-            //get all of the selected checkbox file names and import type
-            $('input.kb-staging-table-body__checkbox-input:checked')
-                .each(function () {
-                    const importType = $(this).attr('data-type');
-                    const importFile = $(this).attr('data-file-name');
-                    stagingAreaViewer.initImportApp(importType, importFile);
-                });
-          
+            Jupyter.narrative.insertBulkImportCell();
+            Jupyter.narrative.hideOverlay();
```

Comment:
For now, can we only launch the bulk importer cell for the target data type? This way we will still be able to import cells for non-target data types.

Reply:
Yup, good idea. I can change that.

Reply:
Ok, done.